app/assets/javascripts/darkswarm/services/stripe_elements.js.coffee

```diff
@@ -35,11 +40,17 @@ Darkswarm.factory 'StripeElements', ($rootScope, Messages) ->
           secrets.cc_type = @mapPaymentMethodsApiCardBrand(response.paymentMethod.card.brand)
           secrets.card = response.paymentMethod.card
           submit()
+      .catch (response) =>
+        # Stripe handles errors in the response above. This code may never be
+        # reached. But if we get here, we want to know.
+        @reportError(response, t("js.stripe_elements.unknown_error_from_stripe"))
+        throw response
 
     reportError: (error, messageForUser) ->
       Messages.error(messageForUser)
       @triggerAngularDigest()
-      console.error(JSON.stringify(error))
+      console.error(error)
+      Bugsnag.notify(new Error(JSON.stringify(error)))
```

Comment:
I think this "new Error" is important. I see "notify received a non error" in bugsnag:
![image](https://user-images.githubusercontent.com/1640378/91845134-8c15d000-ec50-11ea-9357-780c3c974a4f.png)

but I am not sure where in the checkout.js code this one is coming from.
anyway, new Error here will avoid this problem.

Reply:
Yes, I observed the same. The InvalidError is reported without `new Error` and it looks much better with `new Error`. So I'm doing it right, right?